applications/application-life-cycle-management/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -26,5 +26,6 @@ To create applications using the *Developer* perspective ensure that:
 
 * You have xref:../../web-console/web-console.adoc#web-console[logged in to the web console].
 * You are in the xref:../../web-console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have the appropriate link:https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#default-roles_using-rbac[roles and permissions] in a project to create applications and other workloads in {product-title}.
```

Comment:
For internal links, we must follow this format: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references

So, the link should look something like:

xref:../authentication/using-rbac.adoc#default-roles_using-rbac[roles and permissions]